chpl/chpl-api/src/main/java/gov/healthit/chpl/questionableActivity/CertificationResultQuestionableActivityProvider.java

```diff
@@ -144,31 +181,39 @@ public QuestionableActivityCertificationResultDTO checkGapUpdated(
-        
-        List<QuestionableActivityCertificationResultDTO> addedMacras = new ArrayList<QuestionableActivityCertificationResultDTO>();
-        if ((origCertResult.getG2MacraMeasures() == null || origCertResult.getG2MacraMeasures().size() == 0) && 
-                newCertResult.getG2MacraMeasures() != null && newCertResult.getG2MacraMeasures().size() > 0) {
+
+     /**
+      * Check for QA re: addition to G2 MACRA measures.
+      * @param origCertResult original certification criteria result
+      * @param newCertResult new certification criteria result
+      * @return list of added measures
+      */
+      public List<QuestionableActivityCertificationResultDTO> checkG2MacraMeasuresAdded(
+            final CertificationResult origCertResult, final CertificationResult newCertResult) {
+
+        List<QuestionableActivityCertificationResultDTO> addedMacras
+        = new ArrayList<QuestionableActivityCertificationResultDTO>();
+        if ((origCertResult.getG2MacraMeasures() == null || origCertResult.getG2MacraMeasures().size() == 0)
+                 && newCertResult.getG2MacraMeasures() != null && newCertResult.getG2MacraMeasures().size() > 0) {
             //all the newCert G2 macras are "added"
-            for(MacraMeasure newMacra : newCertResult.getG2MacraMeasures()) {
+            for (MacraMeasure newMacra : newCertResult.getG2MacraMeasures()) {
                 QuestionableActivityCertificationResultDTO activity = new QuestionableActivityCertificationResultDTO();
                 activity.setBefore(null);
                 activity.setAfter(newMacra.getAbbreviation());
                 addedMacras.add(activity);
             }
-        } else if (origCertResult.getG2MacraMeasures() != null && origCertResult.getG2MacraMeasures().size() > 0 && 
-                newCertResult.getG2MacraMeasures() != null && newCertResult.getG2MacraMeasures().size() > 0) {
+        } else if (origCertResult.getG2MacraMeasures() != null && origCertResult.getG2MacraMeasures().size() > 0
+                 && newCertResult.getG2MacraMeasures() != null && newCertResult.getG2MacraMeasures().size() > 0) {
             for (MacraMeasure newMacra : newCertResult.getG2MacraMeasures()) {
                 boolean hasMatch = false;
                 for (MacraMeasure origMacra : origCertResult.getG2MacraMeasures()) {
-                    if(newMacra.getId().longValue() == origMacra.getId().longValue()) {
+                    if (newMacra.getId().longValue() == origMacra.getId().longValue()) {
                         hasMatch = true;
                     }
                 }
-                if(!hasMatch) {
-                    QuestionableActivityCertificationResultDTO activity = new QuestionableActivityCertificationResultDTO();
+                if (!hasMatch) {
+                    QuestionableActivityCertificationResultDTO activity
+                    = new QuestionableActivityCertificationResultDTO();
```

Comment:
Should this be indented?  https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent